tests/PHPUnit/Framework/Mock/FakeAccess.php

```diff
@@ -128,7 +128,7 @@ public function checkUserHasViewAccess($idSites)
     public function checkUserHasSomeViewAccess()
     {
         if (!self::$superUser) {
-            if (count(self::$idSitesView) == 0) {
+            if (count(array_merge(self::$idSitesView, self::$idSitesAdmin)) == 0) {
```

Comment:
This was actually a bug in FakeAccess, possible some tests might have behaved wrong because of that in the future